getting-started-vue/11.md

```diff
@@ -0,0 +1,174 @@
+
+```JavaScript
+const platform = require("tns-core-modules/platform");
+const frame = require("tns-core-modules/ui/frame");
+```
+
+<hr data-action="end" />
+
+### Step 2. Improve the looks of the `<ActionBar>` on iOS
+
+Note how the `<ActionBar>` at the top of the screen combines a dark label on a dark background for iOS. You can change this with a simple platform-specific tweak. Use the `platform.isIOS` check to run the configuration change only on iOS. Next, apply a specific iOS status bar setting.
+
+<hr data-action="start" />
+
+#### Action
+
+* **a.** In the `methods` section of the `<script>` block, add a `onPageLoaded` method at the top and create a simple `platform.isIOS` check. You will use it to manage iOS-specific changes as soon as the app screen loads.
+
+```JavaScript
+onPageLoaded: function () {
+    if (platform.isIOS) { 
+
+    }
+},
+```
+
+* **b.** Call this method in the `@loaded` event for the `<Page>` component in your `<template>` block.
+
+```HTML
+<Page class="page" @loaded="onPageLoaded">
+```
+
+* **c.** Use the `frame` module to access the native iOS status bar and change its style.
+
+```JavaScript
+onPageLoaded: function () {
+    if (platform.isIOS) {
+        // Since the ActionBar is a dark color, this snippet of code switches the app to use
+        // white text in the status bar for iOS.
+        const navBar = frame.topmost().ios.controller.navigationBar;
+        navBar.barStyle = UIBarStyle.UIBarStyleBlack;
+    }
+},
+```
+
+* **d.** Tweak the looks of the iOS keyboard by hiding the auto toolbar.
+
+```JavaScript
+IQKeyboardManager.sharedManager().enableAutoToolbar = false;
+```
+
+<hr data-action="end" />
+
+At the end of this step, your `onPageLoaded` method should resemble this sample:
+
+```JavaScript
+onPageLoaded: function () {
+    if (platform.isIOS) {
+        // Since the ActionBar is a dark color, this snippet of code switches the app to use
+        // white text in the status bar for iOS.
+        const navBar = frame.topmost().ios.controller.navigationBar;
+        navBar.barStyle = UIBarStyle.UIBarStyleBlack;
+
+        // Hide the IQKeyboard default toolbar as it might confuse users in this simple example.
+        IQKeyboardManager.sharedManager().enableAutoToolbar = false;
+    }
+},
+```
+
+### Step 3. Import iOS-specific icons
+
+You can add images to use inside your app with a few clicks. Upload iOS-specific icons that you can later use to improve the look of your navigation tabs.
+
+<hr data-action="start" />
+
+#### Action
+
+* **a.** Download [this icon archive](https://github.com/NativeScript/playground-tutorials/blob/master/getting-started-vue/images/icons.zip?raw=true) locally and unpack it.
+
+* **b.** Point to the `app` folder and click the more options button at the far right of its line. Select **Add folder** and name the new folder `images`.
```

Comment:
You might need an image for this one, as I could see readers having trouble finding the three-dot icon.